Makefile

```diff
@@ -198,47 +197,22 @@ env:
 # usage
 #
 
+define help_category
+	printf "\n\033[1m%s\033[0m\n" $(1)
```

Comment:
In this case (mixing awk and makefile functions, see other comment) it would be better if we have variables for the colors so we could configure them without having to change the logic (error prone).